pkg/postgres/roles.go

```diff
@@ -110,6 +111,36 @@ WHERE r."oid" = $1
 	return role, nil
 }
 
+func (c *Client) Grant(ctx context.Context, roleName string, principalName string) error {
+	l := ctxzap.Extract(ctx)
+
+	sanitizedRoleName := pgx.Identifier{roleName}.Sanitize()
```

Comment:
this being input from DisplayNames feels odd to me; kinda wish we had... a connector-specific annotation or somehting

Reply:
Oh yeah. I added a TODO in #13.